src/olympia/shelves/tests/test_forms.py

```diff
@@ -17,44 +14,39 @@ def setUp(self):
         self.criteria_404 = 'sort=users&type=statictheme'
         self.criteria_not_200 = '?sort=user&type=statictheme'
         self.criteria_empty = '?sort=users&type=theme'
-        baseUrl = settings.INTERNAL_SITE_URL
 
         responses.add(
             responses.GET,
-            baseUrl + drf_reverse('v4:addon-search') +
-            self.criteria_sea,
+            reverse_ns('addon-search') + self.criteria_sea,
```

Comment:
We're specifically pinging the v4 api in the forms code above, because that's the version of the api that addons-frontend is currently using.  If you want to use `reverse_ns`, specify `api_version`.